PostHog/PostHogSDK.swift

```diff
@@ -761,7 +812,12 @@ let maxRetryDelay = 30.0
         capture("Application Opened", properties: props)
     }
 
-    @objc func captureAppBackgrounded() {
+    @objc func handleAppDidEnterBackground() {
```

Comment:
Ah yes you're right. I was thinking the background event would cause the timestamp to update, but the user could have `captureApplicationLifecycleEvents` false in which case backgrounding wouldn't cause an event to fire. Will update.